src/HeaderBar/LogoImage.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import css from 'styled-jsx/css'
+
+import { useDataQuery } from '@dhis2/app-runtime'
+import { LogoIconWhite } from '@dhis2/ui-core'
+
+const defaultLogo = css.resolve`
+    svg {
+        height: 25px;
+        width: 27px;
+    }
+`
+
+const query = {
+    customLogo: {
+        resource: 'staticContent/logo_banner',
+    },
+}
+
+export const LogoImage = () => {
+    const { loading, error, data } = useDataQuery(query)
+
+    if (loading) return null
+
+    return (
+        <div>
+            {error ? (
```

Comment:
Is this using a backend bug to detect the lack of a custom logo?  I get a CORS error (incorrectly, since I have `https://*` in my whitelist) when there's no custom logo specified:

 
<img width="839" alt="Screenshot 2019-10-27 01 49 24" src="https://user-images.githubusercontent.com/947888/67625441-0705a180-f85c-11e9-815c-7c47d23b694c.png">


It seems like it wouldn't work in either case to me, since the requested resource (`staticContent/logo_banner`) redirects to the binary `.png` file if there's a custom logo specified, rather than returning a JSON body with `customLogo.images.png` property (dereferenced below).

<img width="1552" alt="Screenshot 2019-10-27 01 48 20" src="https://user-images.githubusercontent.com/947888/67625427-dfaed480-f85b-11e9-955d-2ac2f084da20.png">

Reply:
Aaaandd.... ignore this, was using an older version without Maikel's fix.